src/app/core/market/api/template/template.model.ts

```diff
@@ -179,22 +179,34 @@ export class Template {
   }
 
   get expiredAt(): any {
-    return Object.prototype.toString.call(this.object.ListingItems) === '[object Array]' && this.object.ListingItems.length ?
-    'Expires ' + new DateFormatter(new Date(this.object.ListingItems[0].expiredAt)).dateFormatter(false).substr(0, 16) : '';
+    return  this.checkListingItems ? 'Expires ' + new DateFormatter(
+      new Date(this.object.ListingItems[0].expiredAt)
+      ).dateFormatter(false).substr(0, 16) : '';
   }
 
   get isAboutToExpire(): Boolean {
 
     // 86400000 m seconds in one day.
-    return (this.object.expiredAt > +new Date() && ((this.object.expiredAt - +new Date()) <= 86400000));
+    return (this.object.expiredAt < +new Date() && ((this.object.expiredAt - +new Date()) <= 86400000));
```

Comment:
Was this in error, or intentional? It changes the meaning of the condition to indicate that the item has already expired.

Reply:
@zaSmilingIdiot possibly I was doing some testing and pushed the previous change my bad :( 

Thanks